tests/test_utilities.py

```diff
@@ -156,6 +156,159 @@ def test_check_all_contiguous():
     t2 = time.time()
     print 'time for %s = %.2f'%(funcname(),t2-t1)
 
+def test_interleaveImages():
+    import time
+    t1 = time.time()
+
+    # 1a) With galsim Gaussian
+    g = galsim.Gaussian(sigma=3.7,flux=1000.)
+    gal = galsim.Convolve([g,galsim.Pixel(1.0)])
+    im_list = []
+    offset_list = []
+    n = 2
+    for j in xrange(n):
+        for i in xrange(n):
+            im = galsim.Image(16*n,16*n)
+            offset = galsim.PositionD(-(i+0.5)/n+0.5,-(j+0.5)/n+0.5)
+            offset_list.append(offset)
+            gal.drawImage(image=im,scale=1.0,method='no_pixel',offset=offset)
```

Comment:
Also, rather than convolve by the Pixel above, and then using 'no_pixel' here, you should just draw this the normal way with `g.drawImage(image=im, scale=scale, offset=offset)` and let galsim do the pixel convolution for you.

Then below, you will need to do the explicit convolution by `Pixel(scale)` when rendering the comparison image at `scale=scale/n`.  So still use 'no_pixel' there.